src/libparms/db/parmsdb.cpp

```diff
@@ -1407,20 +1317,6 @@ bool ParmsDb::upgrade(const std::string &fromVersion, const std::string & /*toVe
 
     if (CommonUtility::isVersionLower(dbFromVersionNumber, "3.6.1")) {
         LOG_DEBUG(_logger, "Upgrade < 3.6.1 DB");
-
-        queryFree(CREATE_SELF_RESTARTER_TABLE_ID);
-        ASSERT(queryCreate(CREATE_SELF_RESTARTER_TABLE_ID));
-        if (!queryPrepare(CREATE_SELF_RESTARTER_TABLE_ID, CREATE_SELF_RESTARTER_TABLE, false, errId, error)) {
-            queryFree(CREATE_SELF_RESTARTER_TABLE_ID);
-            return sqlFail(CREATE_SELF_RESTARTER_TABLE_ID, error);
-        }
-
-        if (!queryExec(CREATE_SELF_RESTARTER_TABLE_ID, errId, error)) {
-            queryFree(CREATE_SELF_RESTARTER_TABLE_ID);
-            return sqlFail(CREATE_SELF_RESTARTER_TABLE_ID, error);
-        }
-        queryFree(CREATE_SELF_RESTARTER_TABLE_ID);
-
         createAppState();
```

Comment:
Return code to check

Reply:
[81ea943](https://github.com/Infomaniak/desktop-kDrive/pull/77/commits/81ea943b54acb16d9f335cbdf5ab41dc17c78bfe)